From 47b84144edc75ee2c465ae43d701f3e5d4d00462 Mon Sep 17 00:00:00 2001
From: Daniel Castellanos <luis.daniel.castellanos@intel.com>
Date: Tue, 26 Feb 2019 10:57:50 -0800
Subject: [PATCH] Build fixes

---
 Makefile | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/Makefile b/Makefile
index 7248bbed..bf5f34c4 100644
--- a/Makefile
+++ b/Makefile
@@ -26,7 +26,7 @@ all: build
 
 .PHONY: build
 build:
-	GOBIN=$(BINDIR) $(GO) install $(GOFLAGS) -tags '$(TAGS)' -ldflags '$(LDFLAGS)' k8s.io/helm/cmd/...
+	GOBIN=$(BINDIR) $(GO) install $(GOFLAGS) -tags '$(TAGS)' -ldflags '$(LDFLAGS)' k8s.io/helm/cmd/helm k8s.io/helm/cmd/tiller
 
 # usage: make clean build-cross dist VERSION=v2.0.0-alpha.3
 .PHONY: build-cross
-- 
2.20.1

